### DISCUSSION TITLE
Installation Problems

### DISCUSSION THREAD

**Discussion by DominikCGI** [jas3333/GPT-Helper#34] [General]
After propting any command in the web interface I get this error:

AxiosError: Request failed with status code 429
....

method: null,
      statusCode: 429,
      statusMessage: 'Too Many Requests',

....

data: { error: [Object] }

....

What is the problem

**Comment by jas3333** [comment]
Sounds like an API key issue. If you're on Windows, you probably created a .env.txt instead of just `.env`
